src/XR/features/WebXRHandTracking.ts

```diff
@@ -73,32 +66,38 @@ export interface IWebXRHandTrackingOptions {
          * If enabled, override default physics properties
          */
         physicsProps?: { friction?: number; restitution?: number; impostorType?: number };
+        /**
+         * The utilityLayer scene that contains the 3D UI elements. Passing this in turns on near interactions with the index finger tip
+         */
+        sceneForNearInteraction?: Scene;
+    };
+
+    /**
+     * Configuration object for the hand meshes.
+     */
+    handMeshes?: {
         /**
          * Should the default hand mesh be disabled. In this case, the spheres will be visible (unless set invisible).
          */
-        disableDefaultHandMesh?: boolean;
+        disableDefaultMeshes?: boolean;
         /**
-         * a rigged hand-mesh that will be updated according to the XRHand data provided. This will override the default hand mesh
+         * Rigged hand meshes that will be tracked to the user's hands. This will override the default hand mesh.
          */
-        handMeshes?: {
+        customMeshes?: {

```

Comment:
i totally agree with the change, but it breaks back-compat, as this was introduced in 4.2. If you can add a legacy support in the function itself it would be great.

Reply:
Added legacy support by translating the options object in the constructor